packages/dataviews/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Bug Fixes
+
+- Fix `filterSortAndPaginate` to handle searching fields that have a type of `array`.
```

Comment:
```suggestion
- Fix `filterSortAndPaginate` to handle searching fields that have a type of `array` ([#70785](https://github.com/WordPress/gutenberg/pull/70785)).
```

Usually, changelog entries include a link to the PRs, but it seems to be omitted from more recent ones.